cli/global_state.rs

```diff
@@ -90,6 +90,15 @@ impl ThreadSafeGlobalState {
     let main_module: Option<ModuleSpecifier> = if flags.argv.len() <= 1 {
       None
     } else {
+      // resolve modules for rest of args if present (for fetch)
+      let files_len = flags.argv.len();
+      if files_len > 2 {
+        for i in 2..files_len {
+          let next_specifier = flags.argv[i].clone();
+          let _res = ModuleSpecifier::resolve_url_or_path(&next_specifier);
+        }
+      }
+      // then resolve root
       let root_specifier = flags.argv[1].clone();
       Some(ModuleSpecifier::resolve_url_or_path(&root_specifier)?)
```

Comment:
Does this change actually suffice to add support to multiple files in fetch? I'm a bit confused. I think a test would be useful see the `itest!` commands in `cli/tests/integration_test.rs`

Reply:
That was indeed not enough. Fixed the logic, added integration test - but it currently fails.
Not entirely sure how to test this correctly as it seems like `Compile`/`Download` messages are being suppressed during tests - any hints?

Reply:
Add `check_stderr: true` to `itest!`

Reply:
That helped! Pushing the fix now, thanks!